lib/featuring.js

```diff
@@ -1,6 +1,18 @@
 class Featuring {
 	constructor(colaboradores) {
-		this.colaboradores	= colaboradores;
+		this.colaboradores = colaboradores;
+	}
+
+	equals(other) {
+		if (other instanceof Featuring) {
+			// Comparamos los arrays de colaboradores elemento por elemento
+			if (this.colaboradores.length !== other.colaboradores.length) return false;
+			for (let i = 0; i < this.colaboradores.length; i++) {
+				if (this.colaboradores[i] !== other.colaboradores[i]) return false;
```

Comment:
¿Qué pasa si están en otro orden?

Reply:
Para esto lo que tenéis que hacer es usar estructuras de datos que no dependan del orden, que JS las tiene y abundantes. Además, si lo que estás buscando es una intersección, igual, debes usar una estructura de datos que permita esa operación.

Reply:
Realmente, esa función ya no es necesaria, ya que con la evolución de la implementación no se tienen que hacer comparaciones de las colaboraciones. Para no aumentar la deuda técnica, creo que lo mejor sería eliminar este método.